server/src/weaverbird/backends/mongo_translator/steps/rank.py

```diff
@@ -0,0 +1,91 @@
+from typing import List
+
+from weaverbird.backends.mongo_translator.steps.types import MongoStep
+from weaverbird.pipeline.steps import RankStep
+
+
+def translate_rank(step: RankStep) -> List[MongoStep]:
+    vqb_var_order: MongoStep = {}
+
+    if step.method == 'dense':
```

Comment:
yould you pelase add the comments that were present in the typescript translator here ?